<launch>
  <node name="rec_obj_srv_node" pkg="apc_2016" type="service_recognise_object.py" output="screen" cwd="node"/>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>
  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="image_from_object_recognition"/>
    <param name="autosize" value="true" />
  </node>
</launch>
